cloudinit/distros/__init__.py

```diff
@@ -80,8 +80,11 @@ def _write_network_config(self, settings):
         raise NotImplementedError()
 
     def _supported_write_network_config(self, network_config):
-        priority = util.get_cfg_by_path(
+        priority = self.preferred_renderer_priority
+        cfg_priority = util.get_cfg_by_path(
             self._cfg, ('network', 'renderers'), None)
+        if cfg_priority:   # merged config priority trumps distro defaults
+            priority = cfg_priority
```

Comment:
We want to allow a local cloud-config file to set the the policy.  We have a default policy set in /etc/cloud/cloud.cfg via the distro template.  Let's update that to be netplan, eni, sysconfig.  On release branch we won't add this change to the template file, so we'll need and #SRU Blocker tag there.

It also makes this a much easier patch.

Reply:
Ok dropped this changeset and boiled it down to just config/cloud.cfg.tmpl changes and an SRU_BLOCKER comment. Do you think you need a cloud_test for this case?

Reply:
ok dropped my changeset in favor of just altering config/cloud.cfg.tmpl for ubuntu. This doesn't really have unit tests around it.

Reply:
Let's hold off on the cloud_test for this at the moment since we know this fix is minimal, and desired behavior for Focal. We can put up a separate PR for adding an upgrade/ifupdown cloud_test after we've talked with CPC/Foundations about direction for ifupdown support